File: SPCode/Security/Support/LoginEventArgs.vb
#Region "Imports directives"
Imports System
Imports System.Threading
Imports System.Security.Principal
Imports System.Diagnostics
Imports System.Reflection
Imports System.Drawing
Imports System.Windows.Forms
Imports System.ComponentModel
#End Region

Namespace Security
    Public Class LoginEventArgs
        Inherits EventArgs

        Private m_Authenticated As Boolean

        Public Sub New(ByVal authenticated As Boolean)
            Me.Authenticated = authenticated
        End Sub

        Public Property Authenticated() As Boolean
            Get
                Return m_Authenticated
            End Get
            Friend Set(ByVal value As Boolean)
                m_Authenticated = value
            End Set
        End Property
    End Class
End Namespace

File: SoftLogic.Win/Security/Support/LoginEventArgs.cs
using System.Text.RegularExpressions;
using System.Diagnostics;
using System;
using System.Drawing.Drawing2D;
using System.Windows.Forms;
using System.Collections;
using System.Drawing;
using Microsoft.VisualBasic;
using System.Data.SqlClient;
using System.Data;
using System.Collections.Generic;
using WeifenLuo.WinFormsUI;
using Microsoft.Win32;
using WeifenLuo;
using System.Threading;
using System.Security.Principal;
using System.Reflection;
using System.ComponentModel;

namespace SoftLogik.Win
{
	#region Imports directives
	#endregion
	
	namespace Security
	{
		public class LoginEventArgs : EventArgs
		{
			
			
			private bool m_Authenticated;
			
			public LoginEventArgs(bool authenticated)
			{
				this.Authenticated = authenticated;
			}
			
			public bool Authenticated
			{
				get
				{
					return m_Authenticated;
				}
				internal set
				{
					m_Authenticated = value;
				}
			}
		}
	}
	
}
